gunicorn/arbiter.py

```diff
@@ -230,8 +230,6 @@ def run(self):
             self.halt()
         except HaltServer as inst:
             self.halt(reason=inst.reason, exit_status=inst.exit_status)
-        except SystemExit:
```

Comment:
While I think the change looks fine, leaving it would make exception handling clearer for the reader. I'd add try-except-raise to .pylintrc instead.

Reply:
sounds good